### DISCUSSION TITLE
[button] brick variant

### DISCUSSION THREAD

**Discussion by marionnegp** [RedHat-UX/red-hat-design-system#2073] [Feature discovery]
For the My Red Hat dashboard, something we're exploring is the use of a button with a light gray button that spans the width of its container, which is similar to the [brick variant of CTA](https://ux.redhat.com/elements/call-to-action/guidelines/#variants). It should retain an icon slot, like the other buttons though. Is this a new variant, or can the secondary/tertiary variants be customized?

Example of where a brick-style button might be used:
<img width="385" alt="Screenshot 2024-12-03 at 2 09 41 PM" src="https://github.com/user-attachments/assets/1cf46611-2b3c-44cd-bea1-93f0833d7b05">

**Comment by zeroedin** [comment]
You can customize the brick variant using the css props, quick and dirty example: 

https://codepen.io/zeroedin/pen/QwLjqxe
